controllers/container_images.go

```diff
@@ -0,0 +1,62 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+)
+
+type Images struct {
+	BaremetalOperator            string `json:"baremetalOperator"`
+	BaremetalIronic              string `json:"baremetalIronic"`
+	BaremetalIronicInspector     string `json:"baremetalIronicInspector"`
+	BaremetalIpaDownloader       string `json:"baremetalIpaDownloader"`
+	BaremetalMachineOsDownloader string `json:"baremetalMachineOsDownloader"`
+	BaremetalStaticIpManager     string `json:"baremetalStaticIpManager"`
+}
+
```

Comment:
why have 2 structs with the same contents? just use "Images"

Reply:
Done.

Reply:
Looks like this commit still has both structs

Reply:
Could you please check again? Seems like a timing issue with comment and code refresh

Reply:
Yes sorry. Looks like I had an older cache. This looks good now. Thanks.